### PR TITLE
Prefs [Task][27953] Create UI for Statistics screen

### PR DESCRIPTION
**Pull request by NguyenTaTrung**
## Related Tickets
- [#Task_27953: Create UI for Statistics screen](https://edu-redmine.sun-asterisk.vn/issues/27953)

## WHAT
- Create UI use:
+ Bottom Navigation show 3 tabs
+ RadioGroup, RadioButton
+ ImageButton notification, ImageView
+ TextView to show information about covid19 (time update, infected, new infected,...)
+ Custom button, layout and select color

## Evidence (Screenshot or Video)
statistic

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Trainer reivew (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the source code have end-line errors? | | NO |<li>- [x] no</li>|<li>- [x] no</li>  
Conventions | No hard coding, use constants/configuration values | | YES |<li>- [x] yes</li>|<li>- [x] yes</li>  
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [x] yes</li>|<li>- [x] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [x] yes</li>|<li>- [x] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [x] yes</li>|<li>- [x] yes</li>

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,6 +31,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.core:core-ktx:1.3.1'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'com.android.support:design:27.1.0'
```
Comment (aug16vcc) [inline app/build.gradle]: Chuyển sang dùng `androidx` artifact
B1: nâng `com.android.support:design` lên `v28` 
B2: sync gradle -> build project nếu có lỗi thì fix
B3: đổi `com.android.support:design` thành `com.google.android.material:material:1.0.0-rc01`
B4: làm giống B2
B5: cập nhật version `com.google.android.material` lên bản mới nhất
B6: làm giống B2
https://developer.android.com/jetpack/androidx/migrate/artifact-mappings

File: app/src/main/res/layout/statistics_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,337 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorPrimary">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline"
```
Comment (aug16vcc) [inline app/src/main/res/layout/statistics_fragment.xml]: Tên rõ nghĩa hơn, ví dụ: `guidelineTop`

File: app/src/main/res/layout/statistics_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@drawable/view_bottom_background"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/viewTop" />
+
+    <RadioGroup
+        android:id="@+id/toggle"
```
Comment (aug16vcc) [inline app/src/main/res/layout/statistics_fragment.xml]: Nên đặt tên rõ nghĩa hơn

File: app/build.gradle (hunk 1 above)
Comment (NguyenTaTrung) [reply app/build.gradle]: Ok tks chú t đang fix lại

File: app/src/main/res/layout/statistics_fragment.xml [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+        android:id="@+id/guideline"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintEnd_toStartOf="@+id/imageViewDoctor"
+        app:layout_constraintGuide_percent="0.3"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <View
+        android:id="@+id/viewTop"
```
Comment (aug16vcc) [inline app/src/main/res/layout/statistics_fragment.xml:18]: Nếu dùng `findViewById` thì đặt tên kiểu `snake` còn nếu dùng `android kotlin extension` thì để `camel` như này nhé

File: app/src/main/res/layout/statistics_fragment.xml [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/colorRed"
+        app:layout_constraintBottom_toTopOf="@+id/textViewTotalDeath"
+        app:layout_constraintEnd_toEndOf="@+id/textViewDeath"
+        app:layout_constraintStart_toStartOf="@+id/textViewDeath"
+        app:layout_constraintTop_toBottomOf="@+id/buttonDeath" />
+
+    <TextView
+        android:id="@+id/title"
```
Comment (aug16vcc) [inline app/src/main/res/layout/statistics_fragment.xml]: Tên tối nghĩa

**Review by aug16vcc — APPROVED** [review]

**Review by tuannm-1276 — APPROVED** [review]

**Pull request merged**